src/core/utils/index.js

```diff
@@ -366,3 +366,12 @@ export function addonHasVersionHistory(addon) {
     ADDON_TYPE_THEME,
   ].includes(addon.type);
 }
+
+export const getRelativePath = (filename) => {
```

Comment:
This should live in `errorHandler` and be named something more like `normalizeFileNameId()` because I don't want someone to think they can use it to get relative paths :)

Reply:
That's right, I was not sure about moving it or not, to be honest. I told myself "there is 99% chance someone will want to get a relative path of a source file and 1% will be @kumar303 telling me to put it in the `errorHandler`".